src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -587,8 +586,13 @@ private void ParsePatchElement(XElement patchElement, out bool upToDate, out boo
                 var contentsAttribute = element.Attribute(ContentAttributeName);
                 if (contentsAttribute == null)
                 {
-                    _service._reportAndSwallowException(
```

Comment:
❓ Would it break bucketing to just modify the implementation of `_reportAndSwallowException`?